.idea/inspectionProfiles/profiles_settings.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Whoops. Looks like an phpstorm file got added to this commit somehow. Remove please?